lib/data_pull.rb

```diff
@@ -263,4 +266,48 @@ def run(args:, config:)
       )
     end
   end
+
+  class ProfileSummary
+    def run(args:, config:)
+      uuids = args
+
+      users = User.includes(:profiles).where(uuid: uuids).order(:uuid)
+
+      table = []
+      table << %w[uuid profile_id status activated_timestamp disabled_reason
+                  gpo_verification_pending_timestamp fraud_review_pending_timestamp
+                  fraud_rejection_timestamp]
```

Comment:
let's have this on separate lines, I think it's easier to follow
```suggestion
      table << %w[
        uuid
        profile_id
        status
        activated_timestamp
        disabled_reason
        gpo_verification_pending_timestamp
        fraud_review_pending_timestamp
        fraud_rejection_timestamp
      ]
```